<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Plymouth</class>
 <widget class="QDialog" name="Plymouth">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>715</width>
    <height>580</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>715</width>
    <height>580</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>715</width>
    <height>580</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Plymouth Theme</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>661</width>
     <height>141</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="remastersys.qrc">:/images/remastersys-large.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>391</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Select Default Plymouth Theme for System</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>520</y>
     <width>671</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="selectBtn">
      <property name="maximumSize">
       <size>
        <width>163</width>
        <height>35</height>
       </size>
      </property>
      <property name="text">
       <string>Select</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="newBtn">
      <property name="maximumSize">
       <size>
        <width>163</width>
        <height>35</height>
       </size>
      </property>
      <property name="text">
       <string>New</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="previewBtn">
      <property name="maximumSize">
       <size>
        <width>163</width>
        <height>35</height>
       </size>
      </property>
      <property name="text">
       <string>Preview</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="backBtn">
      <property name="maximumSize">
       <size>
        <width>163</width>
        <height>35</height>
       </size>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>671</width>
     <height>331</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="remastersys.qrc"/>
 </resources>
 <connections/>
</ui>
